nerdlets/pathpoint-nerdlet/services/DataManager.js

```diff
@@ -2322,6 +2359,7 @@ export default class DataManager {
               type: 'PRC',
               query: query.query,
               timeout: query_timeout,
+              measure_time: query_measure_time,
```

Comment:
Without these changes it was impossible for us to save an update to the since clause